core/src/main/java/io/parsingdata/metal/Shorthand.java

```diff
@@ -124,6 +126,8 @@ private Shorthand() {}
 
     public static BinaryValueExpression add(final ValueExpression left, final ValueExpression right) { return new Add(left, right); }
     public static BinaryValueExpression div(final ValueExpression left, final ValueExpression right) { return new Div(left, right); }
+    public static BinaryValueExpression divCeil(final ValueExpression left, final ValueExpression right) { return new DivCeil(left, right); }
+    public static BinaryValueExpression divFloor(final ValueExpression left, final ValueExpression right) { return new DivFloor(left, right); }
```

Comment:
Why `divCeil` and `divFloor`, and not just `ceil` and `floor`? That way we can just combine `ceil` and `floor` with other `BinaryValueExpression`s (e.g. `ceil(mul(...))` and `ceil(add(...))`).